### PR TITLE
Adding gedit 3.2.6-3

### PR DESCRIPTION
**Pull request by SeanSith**
Adding gedit to close issue #11514 also fixing caveat stated in
previous PR #10261.

### CODE REVIEW

**Comment by SeanSith** [comment]
I apologize for the postflight block. I'm still learning Ruby and this seemed relatively painless. I'm open to suggestions on how to improve it.

File: Casks/gedit.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+  depends_on :formula => 'libxml2'
+
+  app 'gedit.app'
+
+  postflight do
+  	library = Dir.glob("#{`brew --cellar`.chomp}/libxml2/**/libxml2.2.dylib").first
+  	system 'cp', '-f', library, "#{staged_path}/gedit.app/Contents/Resources/lib/"
+  end
+end
```
Comment (vitorgalvao) [inline Casks/gedit.rb]: It has a “no newline at end of file” warning. Those are [actually meaningful](http://unix.stackexchange.com/a/18789).

File: Casks/gedit.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  name 'gedit'
+  homepage 'https://wiki.gnome.org/Apps/Gedit'
+  license :gpl
+
+  depends_on :formula => 'libxml2'
+
+  app 'gedit.app'
+
+  postflight do
+  	library = Dir.glob("#{`brew --cellar`.chomp}/libxml2/**/libxml2.2.dylib").first
```
Comment (vitorgalvao) [inline Casks/gedit.rb]: Indentation should be two spaces.

**Comment by vitorgalvao** [comment]
Looks good, code-wise. Tested and working?

**Comment by SeanSith** [comment]
Code tested and working. I was mostly concerned with having multiple versions of libxml2 installed and hence my use of the 'first' method being inappropriate. Formatting issues fixed, as well.

**Pull request merged**